CONTRIBUTING.md

```diff
@@ -111,7 +111,13 @@ The documentation is generated in the `build/docs` folder.
 
 To fully benefit the ESLint and Jest testing frameworks, you must properly set up your IDE.
 
-##### IntelliJ
+##### Intellij
+
+*EditorConfig* (https://www.jetbrains.com/help/idea/configuring-code-style.html#editorconfig)
+
+Go to `File` -> `Settings` and `Editor` --> `Code Style`, then tick the `Enable EditorConfig support`
```

Comment:
I don't have the section **Setting** in **File**, only **Manage IDE Settings**. Is it that ?

<img width="665" alt="image" src="https://user-images.githubusercontent.com/4921914/85391218-2cad8c80-b54a-11ea-8482-17de6935be60.png">

Reply:
Apparently, on macOS, the settings are accessible in a different way, through **IntelliJ IDEA | Preferences**
See https://www.jetbrains.com/help/idea/settings-preferences-dialog.html
If you confirm this, we should update the contributing guide

Reply:
Yes, it's that :)

Reply:
updated